distributed/deploy/local.py

```diff
@@ -30,6 +30,8 @@ class LocalCluster(SpecCluster):
     ----------
     n_workers: int
         Number of workers to start
+    memory_limit: str
+        Sets the memory limit per worker. Cannot exceed the total memory available.
```

Comment:
```suggestion
        Sets the memory limit *per worker*.
        
        Note that the limit will only be enforced when ``processes=True``, and the limit is only
        enforced on a best-effort basis—it's still possible for workers to exceed this limit.
```
Technically, exceeding the total memory available is okay: https://github.com/dask/distributed/blob/caf51895f63de4338a5a0c7862ecba0b8f7eb002/distributed/worker_memory.py#L404

And there are some issues with the implementation of the memory limit https://github.com/dask/distributed/issues/6177, so I'd also like to document that.